server/games/typedefs.py

```diff
@@ -20,6 +20,12 @@ class Victory(Enum):
     ERADICATION = 2
     SANDBOX = 3
 
+@unique
+class GameType(Enum):
+    NONE = 0
```

Comment:
Do we need `NONE`?

Reply:
No, in my first iteration i added None because i did not know when this would be used, so i'll remove this.